boudicca.base/publisher-event-html/src/main/resources/templates/events/event.hbs

```diff
@@ -8,14 +8,14 @@
             </svg>
         {{/if}}
         {{#if event.url}}
-            <a class="anchor-to-eventpage" href="{{event.url}}" target="_blank" aria-describedby="Zur Eventseite von {{event.name}} (neues Fenster wird geöffnet)">
-                Zur Eventseite
-            </a>
+            <button class="anchor-to-event">
+                Weitere Einzelheiten
```

Comment:
Maybe we should use "Mehr Informationen" as text here

Reply:
yes, I agree.

Reply:
done.